### PR TITLE
Javascript/Javascript2/week2

### PR DESCRIPTION
**Pull request by dinythomas89**

### CODE REVIEW

File: javascript/javascript2/week2/main.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+//Searching a product 
+const searchBox = document.querySelector('.search input');
+searchBox.addEventListener('keyup', myFunction);
+function myFunction() {
+    const searchBoxValue = searchBox.value.toLowerCase();
+    const filteredArray = products.filter(product => {
+        if (product.name.toLowerCase().includes(searchBoxValue)) {
+            productsUl.innerHTML = '';
+            return product.name;
+        }
```
Comment (cameronjdavis) [inline javascript/javascript2/week2/main.js]: This is only a suggestion. An alternative is to return early.
```suggestion
        if (!product.name.toLowerCase().includes(searchBoxValue)) {
            return;
        }
        
        productsUl.innerHTML = '';
        return product.name;
```

File: javascript/javascript2/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    products.forEach(product => {
+        productLi = document.createElement('li');
+        productName = document.createElement('h2');
+        productPrice = document.createElement('p');
+        productRating = document.createElement('p');
+
+        productName.innerHTML = product.name;
+        productPrice.innerHTML = `Price: ${product.price}`;
+        productRating.innerHTML = `Rating: ${product.rating}`;
+
```
Comment (cameronjdavis) [inline javascript/javascript2/week2/main.js:19]: Get rid of extra blank lines. Less lines means less bugs.

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week2/main.js (hunk 1 above)
Comment (dinythomas89) [reply javascript/javascript2/week2/main.js]: Thanks Cameron. did as you mentioned.

File: javascript/javascript2/week2/main.js (hunk 2 above)
Comment (dinythomas89) [reply javascript/javascript2/week2/main.js:19]: removed the extra lines from the code

**Pull request merged**